publish/publish.proj

```diff
@@ -185,6 +187,23 @@
     </ItemGroup>
     <Error Condition="'@(_DownloadedSymbolsPackages)' != '' and '$(NuGetSymbolsFeedUrl)' == ''" Text="Missing required property NuGetSymbolsFeedUrl" />
     
+  </Target>
+
+  <Target Name="GetPackagesToSign">
+    <ItemGroup>
+      <FilesToSign Include="@(_DownloadedStandardPackages)">
+        <Authenticode>NuGet</Authenticode>
+      </FilesToSign>
+    </ItemGroup>
+    <Message Importance="High" Text="Attempting to sign package '%(FilesToSign.Identity)'" />
```

Comment:
Might want to add an error here if there are no files to sign.

Reply:
A concern might be if there could be a servicing release where no packages are built in the first place, but I don't know why that would happen.

Reply:
Trying to think of an edge case, maybe a Debian package itself (or something else this repo builds) could need some servicing fix that doesn't affect the shared framework?

Seems like enough of a stretch that catching "no packages downloaded" problems is more useful than avoiding a respin in some weird servicing scenario down the line.

Reply:
Yes, given the history of suddenly having unsigned official builds, we've been adding these kinds of checks into the signing in order to try to catch issues sooner.

e.g. https://github.com/dotnet/core-setup/blob/3b80e812b14cd7bbdc87c6e4916523bed0de9a0b/signing/sign.proj#L54

Reply:
Yeah I think it is reasonable to assume we have at least one package to sign.